code/game/data_huds.dm

```diff
@@ -222,6 +223,11 @@
 		else if(sanguinal_amount > 0)
 			xeno_reagent.overlays += sanguinal_image
 
+		if(lightsanguinal_amount > 10)
```

Comment:
no, look at how neurotox_amount is calculated

Reply:
done